app/packages/playback/src/views/TimelineTrack/TimelineTrack.module.css

```
@@ -65,28 +69,71 @@
+.intervalBar:hover {
+  box-shadow: inset 0 0 0 100px rgba(255, 255, 255, 0.12);
+}
+
+/* Each edge handle straddles the bar boundary: `--resize-inset` inside
+   (preserving the move-vs-resize split within the bar) plus a `--resize-buffer`
+   invisible buffer hanging *past* the edge so you can grab right at or just
+   beyond the endpoint. */
+.resizeHandle {
+  --resize-inset: 6px;
+  --resize-buffer: 6px;
+  position: absolute;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)